docs/dev/env_devscripts.md

```diff
@@ -32,7 +32,7 @@ Go to https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/, clic
 right, copy the login command, extract the token from the command and use it to set `CI_TOKEN` in `config_$USER.sh`.
 
 #### Get pull secret
-Convert the secret obtained from [cloud.openshift.com](https://cloud.redhat.com/openshift/install/pull-secret), originally in text format, into JSON format and save it as `pull_secret,json`. 
+Convert the secret obtained from [cloud.openshift.com](https://cloud.redhat.com/openshift/install/pull-secret), originally in text format, into JSON format and save it as `pull_secret.json`. 
```

Comment:
Not related to this change, but I just wonder what did you mean by 'conversion to JSON'? I.e. which operation exactly did you have to make? I've just tried to use the 'Copy pull secret' button, and the result seems the same as the downloaded file:/

Reply:
Correct, but I originally clicked on "Download pull secret", resulting in txt format, thus had to convert. 
Maybe I should tweak the sentence to reflect the fact that we should either use "copy pull secret"(json already) or if using the "Download pull secret"(txt) option you'd to convert to json.